pkg/alexa/gen/model.go

```diff
@@ -247,20 +264,26 @@ func (t *ModelTypeBuilder) WithValuesName(valuesName string) *ModelTypeBuilder {
 }
 
 func (t *ModelTypeBuilder) WithLocaleValues(locale string, values []string) *ModelTypeBuilder {
-	loc, _ := t.registry.Resolve(locale)
+	loc, err := t.registry.Resolve(locale)
+	if err != nil {
+		return t
+	}
 	if t.valuesName == "" {
 		t.valuesName = t.name + l10n.KeyPostfixValues
 	}
 	loc.Set(t.valuesName, values)
```

Comment:
Why does this set values in the locale. You could potentially be overwriting data that you dont know about. Is there a particular reason to do this?

Reply:
fixed